conu/backend/docker/image.py

```diff
@@ -44,19 +46,37 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return super(DockerImageFS, self).__exit__(exc_type, exc_val, exc_tb)
 
 
+class DockerImagePullPolicy(enum.Enum):
+    NEVER = 0
+    IF_NOT_PRESENT = 1
+    ALWAYS = 2
+
+
 class DockerImage(Image):
     """
     Utility functions for docker images.
     """
-    def __init__(self, repository, tag="latest"):
+    def __init__(self, repository, tag="latest", pull_policy=DockerImagePullPolicy.IF_NOT_PRESENT):
```

Comment:
Not important, but - Is there a possibility to early catch this bad example of usage?
```
>>> i = DockerBackend.ImageClass('fedora', DockerImagePullPolicy.ALWAYS)
08:33:00.186 image.py          DEBUG  pull policy set to 'if_not_present' and image is not present, pulling the image
08:33:00.215 image.py          DEBUG  b'{"status":"Trying to pull repository registry.fedoraproject.org/fedora ... "}\r\n'
08:33:02.399 image.py          DEBUG  b'{"status":"Trying to pull repository registry.access.redhat.com/fedora ... "}\r\n'
08:33:04.454 image.py          DEBUG  b'{"status":"Trying to pull repository docker.io/library/fedora ... "}\r\n'
08:33:06.330 image.py          DEBUG  b'{"errorDetail":{"message":"manifest for docker.io/fedora:DockerImagePullPolicy.ALWAYS not found"},"error":"manifest for docker.io/fedora:DockerImagePullPolicy.ALWAYS not found"}\r\n'
```